pinot-common/src/main/java/org/apache/pinot/common/utils/tls/RenewableTlsUtils.java

```diff
@@ -275,6 +291,44 @@ static void reloadSslFactoryWhenFileStoreChanges(SSLFactory baseSslFactory,
     }
   }
 
+  @VisibleForTesting
+  static void reloadSslFactory(SSLFactory baseSslFactory,
```

Comment:
This code is in the FileWatcher block as well. Let's refactor and use the the same method to update the certs from FileWatcher and the scheduled thread. Consider synchronizing the method to avoid concurrent updates from two threads.

Reply:
Done